src/Components/Nav/Components/Category.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import './Category.scss';
+
+class Category extends Component {
+  constructor() {
+    super();
+    this.state = {
+      categoryList: [],
+      isSubOpen: false,
+      categoryIdx: '',
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/category.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(categoryData => {
+        this.setState({
+          categoryList: categoryData,
+        });
```

Comment:
```suggestion
      .then(categoryList => this.setState({ categoryList });
```
- 객체의 key와 value에 들어오는 변수명이 같을 경우 생략해줄 수 있습니다 (객체 단축 속성명)

Reply:
넵